src/main/java/Jerry/Event.java

```diff
@@ -0,0 +1,53 @@
+package Jerry;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents an event task in the chatbot application. An event task is a task that occurs at a specific time or over a specific period.
+ */
+public class Event extends Task {
+    private LocalDateTime from;
+    private LocalDateTime to;
+
+    /**
+     * Creates an event task with the specified description, start time, and end time.
+     *
+     * @param description The description of the event task.
+     * @param from The start time of the event.
+     * @param to The end time of the event.
+     */
+
+    public Event(String description, String from, String to) {
+        super(description);
+        try {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+            this.from = LocalDateTime.parse(from, formatter);
+            this.to = LocalDateTime.parse(to, formatter);
+        } catch (DateTimeParseException e) {
+            System.out.println("Invalid date/time format. Please use yyyy-MM-dd HH:mm.");
+        }
+    }
+
+    @Override
+    public String getTypeIcon() {
+        return "E";
+    }
+
+    @Override
+    public String toString() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+        return super.toString() + " (from: " + from.format(formatter) + " to: " + to.format(formatter) + ")";
+    }
+
+    @Override
+    public String toFileFormat() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+        return "E | " + (isDone ? 1 : 0) + " | " + description + " | " + from.format(formatter) + " | " + to.format(formatter);
```

Comment:
Should your line length be longer than 120 chars?
```suggestion
        return "E | " + (isDone ? 1 : 0) + " | " + description + " | " 
                + from.format(formatter) + " | " + to.format(formatter);
```